src/sources/tt.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "url": "http://ei.weblakes.com/RTTPublic/DshBrdStation/TSChartData?aStationKey=$station&aPeriodKey=1&aParameterKey=$parameter&fChartType=timeseries&MaxLegendEntries=12&IsStack=False&IsCategory=False&fTSShape=line&Context_Bootstrap_Flag=true&_=",
+        "adapter": "trinidadtobago",
+        "name": "Trinidad and Tobago",
```

Comment:
more descriptive provider name

Reply:
I updated these for you

Reply:
sorry lets revert back to name : "trinidadtobago"